integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/BenchUtils.scala

```diff
@@ -636,6 +657,11 @@ case class WriteCsv(
     mode: SaveMode,
     writeOptions: Map[String, String]) extends ResultsAction
 
+case class WriteOrc(
```

Comment:
I don't see a corresponding place where this case class is used in a pattern match.  Looks like `runBench` needs to be updated?

Reply:
Thanks. I've just pushed that.

Reply:
Has this been tested?

Reply:
Sorry, I was testing in parallel and should have waited before pushing. There was one other match that needed to be updated. I have tested with TPC-DS writing to Orc now.